Q4 planning: Feature success (#6785)
PostHog/posthog.com: contents/handbook/small-teams/feature-success/objectives.mdx

As always, reliability is the #1 unwritten goal: Making sure feature flags are reliable trumps every other objective.

> ### Objective: Get 5 reference customers for surveys

Reviewer: ```suggestion
### Objective: Get 5 reference customers for Surveys
```

Do we capitalize features? @joethreepwood

Reviewer 2: All features should be written in sentence case

Last quarter, we built out surveys which has been very well received. This quarter, we're doubling down on surveys.

We will

> - Launch pricing for surveys, make it its own standalone product.

Reviewer: ```suggestion
- Launch pricing for surveys and make it its own standalone product.
```

> - Build out features critical to paying users, like templates, better result visualisation, data export, and potentially event targeting for surveys.

Reviewer: Build out features critical to surveys like templates, multiple questions and logic, better result visualizations, data export, and event targeting.

> - Build out features critical to paying users, like templates, better result visualisation, data export, and potentially event targeting for surveys.

Author: ```suggestion
- Build out features critical to surveys like templates, multiple questions and logic, better result visualizations, data export, and event targeting.
```

> - ???

Author: @liyiy is there anything else to add to this list that we already know we'll work on?

Reviewer: Multiple questions and logic for that

> - ???

Author: ```suggestion
```
